### DISCUSSION TITLE
DNS/Nameserver issue.

### DISCUSSION THREAD

**Discussion by Booklifewealth** [vercel/community#4859] [Help] [answered]
### Summary

Hi,

Im trying to host single page website for my custom domain via GitHub. The website(hosting/DNS) is not working. 

1. I created a new project.
2. linked my GitHub repository and imported the code base. (Tech Stack is - HTML and CSS)
3. I added my custom domain to the project.

Im getting following message at domain page of the prject.

"Another Vercel account is using this domain. Please set the following TXT record on _vercel.domain.com to use domain.com in this project. If you wish to [transfer](#move-button) the entire domain, this will need to be done by the current domain owner. Once the verification is completed and the domain is successfully configured, the TXT record can be removed."

I have been added the TXT record for the domain at registrar end.

attached screenshot for your reference.

![DNS record](https://github.com/vercel/community/assets/123085855/e1d6bc9e-7756-40b3-a0dd-a2fced531797)

### Steps to Reproduce

Some times Im getting "**Page not found**" message and some times **404 Error**

Please fins the attached screen shot.

**Comment by Booklifewealth** [comment]
The website is not loading, following message is displayed.

![Screenshot from 2023-11-15 11-38-11](https://github.com/vercel/community/assets/123085855/deb50fe6-3026-48de-9cb7-f5dbd2b4c9fd)

**Comment by amyegan** [comment]
Hi @Booklifewealth! We have a domain troubleshooting doc that covers common scenarios and things to check. You can find it here: https://vercel.com/docs/projects/domains/troubleshooting

If you'd like help investigating this, please let me know the domain so I can check on it for you. 🙏

**Reply by Booklifewealth** [reply]
Thanks for your response.

**Reply by amyegan** [reply]
I don't see the necessary TXT record on `_vercel.cbdcgpt.com`. Can you double check that the record was added with the necessary value?

https://www.whatsmydns.net/#TXT/_vercel.cbdcgpt.com

**Reply by Booklifewealth** [reply]
I have added to my registrar. we can see it in my first screeshot. 


please can you re check once

**Comment by Booklifewealth** [comment]
Hi  amyegan,

Please can you share step to invoke "A record" to add my domain to the project. 

I added "A Records" to my other domains its working and  hosted successfully.

**Reply by amyegan** [reply, marked as answer]
To add a domain, you can use A and CNAME records. If another project is already using the domain, you need [verify domain access](https://vercel.com/docs/projects/domains/add-a-domain#verify-domain-access) by adding a TXT record on the `_vercel` subdomain with the provided value from your project domain settings. 

In your most recent screenshot, it looks like you have the value assigned to the `www` subdomain instead. If you switch that to `_vercel`, the domain can be verified. The other TXT can be removed.

https://vercel.com/docs/projects/domains/add-a-domain

**Comment by Booklifewealth** [comment]
Thanks for the details.

Its working now.

Site is up and running!
